eZ/Publish/Core/REST/Client/ContentTypeService.php

```diff
@@ -360,11 +360,12 @@ protected function isErrorResponse(Message $response)
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException If the content type draft owned by the current user can not be found
      *
-     * @param mixed $contentTypeId
+     * @param int $contentTypeId
+     * @param bool $ignoreOwnership not supported yet
      *
      * @return \eZ\Publish\API\Repository\Values\ContentType\ContentTypeDraft
      */
-    public function loadContentTypeDraft($contentTypeId)
+    public function loadContentTypeDraft($contentTypeId, bool $ignoreOwnership=false)
```

Comment:
CS Nitpick: other signatures have spaces around `=`, so this one should too (though CS fixer doesn't complain).